wasmapi/go/syscall.go

```diff
@@ -56,15 +56,13 @@ type SyscallDeclaration struct {
 }
 
 func GetSyscallName(id uint16) (string, error) {
-	ptr := bufPtr(getSyscallName(uint32(id)))
-	if ptr == 0 {
+	dst := make([]byte, 64)
```

Comment:
Define a constant `maxSyscallLength = 64` somewhere.

From [syscalls.csv](https://github.com/seccomp/libseccomp/blob/main/src/syscalls.csv), the max length is 28 so using 64 is fine.
```
cut -d, -f1 syscalls.csv  |awk '{print length($0)" "$0}'|sort -n
```